interfaces/builtin/bluez.go

```diff
@@ -91,7 +96,7 @@ var bluezConnectedPlugAppArmor = []byte(`
 # Allow all access to bluez service
 dbus (receive, send)
     bus=system
-    peer=(label=bluez5_bluez_*),
+    peer=(label=snap.bluez.*),
```

Comment:
From what I got the agreement was that we go with this until we have a proper mechanism in place (which @zyga and @jdstrand are working on) to replace the peer label with the correct label from the connected plug.